src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,25 @@
+import static duke.common.Formats.DT_INPUT_FORMAT;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+
+import duke.tasks.Deadline;
+
+public class DeadlineTest {
+    @Test
+    public void outputTest1() {
```

Comment:
Minor edit: Maybe you can consider changing the name of the test methods to match the format just to make it a little clearer:  
featureUnderTest_testScenario_expectedBehavior()
Source: https://se-education.org/guides/conventions/java/intermediate.html#naming